### PR TITLE
Films

### PR DESCRIPTION
**Pull request by Vakulaaa**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,5 @@ venv
 
 __pycache__
 .idea
+
+python
```
Comment (y-u-n-ii-a) [inline .gitignore:7]: What is it?)

File: apps/authentication/migrations/0001_initial.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
- # Generated by Django 4.2.8 on 2024-01-31 12:25
+# Generated by Django 4.2.8 on 2024-02-03 11:30
```
Comment (y-u-n-ii-a) [inline apps/authentication/migrations/0001_initial.py:1]: Just for the future: if migrations were already merged to the common branch never delete them

Comment (y-u-n-ii-a) [reply apps/authentication/migrations/0001_initial.py:1]: But for now, for our project it's ok

File: films/apps.py [hunk 3]
```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
+class FilmsConfig(AppConfig):
+    default_auto_field = "django.db.models.BigAutoField"
+    name = "films"
```
Comment (y-u-n-ii-a) [inline films/apps.py:6]: You should fix the name to "apps.films"

File: films/models.py [hunk 4]
```diff
@@ -0,0 +1,5 @@
+from django.db import models
+
+
+class Movie:
```
Comment (y-u-n-ii-a) [inline films/models.py:4]: All models must inherit `models.Model`

File: films/models.py [hunk 5]
```diff
@@ -0,0 +1,5 @@
...
+    title = models.CharField("title", null=True)
```
Comment (y-u-n-ii-a) [inline films/models.py:5]: Look, in the `User` model you have `Birthday` (in description). You should follow the same style and fix the description here:
```suggestion
    title = models.CharField("Title", null=True)
```

Moreover, the description by default will be the same, so you can just remove it

Comment (y-u-n-ii-a) [inline films/models.py:5]: Make it not nullable

File: movies/settings.py [hunk 6]
```diff
@@ -41,7 +41,7 @@
     # Installed
     "rest_framework",
     # Custom
-    "apps.authentication"
+    "apps.authentication",
```
Comment (y-u-n-ii-a) [inline movies/settings.py:44]: Don't forget to add new apps (`films`)

File: movies/settings.py [hunk 7]
```diff
@@ -139,3 +139,6 @@
 
 # User
 AUTH_USER_MODEL = "authentication.User"
+
+# Movie
+AUTH_MOVIE_MODEL = "authentication.Movie"
```
Comment (y-u-n-ii-a) [inline movies/settings.py:144]: Remove it :)

**Review by y-u-n-ii-a — CHANGES_REQUESTED** [review]
Looks not bad, but some changes needed

File: .gitignore (hunk 1 above)
Comment (Vakulaaa) [reply .gitignore]: fixed

File: apps/authentication/migrations/0001_initial.py (hunk 2 above)
Comment (Vakulaaa) [reply apps/authentication/migrations/0001_initial.py:1]: ok, thanks

File: films/apps.py (hunk 3 above)
Comment (Vakulaaa) [reply films/apps.py]: fixed

Comment (Vakulaaa) [reply films/apps.py]: added

File: films/models.py (hunk 4 above)
Comment (Vakulaaa) [reply films/models.py]: fixed

File: films/models.py (hunk 5 above)
Comment (Vakulaaa) [reply films/models.py]: maked

File: movies/settings.py (hunk 6 above)
Comment (Vakulaaa) [reply movies/settings.py:44]: ok

File: movies/settings.py (hunk 7 above)
Comment (Vakulaaa) [reply movies/settings.py]: removed)

**Review by y-u-n-ii-a — APPROVED** [review]
Looks good to me!

**Pull request merged**